Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -1022,12 +1035,17 @@ static RetainPtr<CTFontRef> fontWithFamily(FontDatabase& fontDatabase, const Ato
     if (family.isEmpty())
         return nullptr;
 
-    if (auto fontDescriptor = fontDescriptorWithFamilySpecialCase(family, fontDescription, size, fontDescription.shouldAllowUserInstalledFonts())) {
-        auto font = createFontForInstalledFonts(fontDescriptor.get(), size, fontDescription.shouldAllowUserInstalledFonts());
-        return preparePlatformFont(font.get(), fontDescription, fontCreationContext, true);
+    if (auto unrealizedFont = fontDescriptorWithFamilySpecialCase(family, fontDescription, size, fontDescription.shouldAllowUserInstalledFonts())) {
+        unrealizedFont->setSize(size);
+        unrealizedFont->modify([allowUserInstalledFonts = fontDescription.shouldAllowUserInstalledFonts()](CFMutableDictionaryRef attributes) {
```

Comment:
I find it a bit weird to use a captured variable to store the result of this expression; WDYT about capturing the `fontDescription` reference, and calling `shouldAllowUserInstalledFonts()` inside the lambda body? Or just capture `[&]` which I think is very common in the codebase.